templates/mdnext-overlays/src/hooks/useChatListener.js

```diff
@@ -21,7 +21,7 @@ export default function useChatListener({ channel, commands = {} }) {
     },
     channels: [channel],
     identity: {
-      username: 'domitriusclark',
+      username: process.env.CHANNEL,
```

Comment:
will this always be the same as the channel property?

Reply:
yeah it should be

Reply:
cool i updated this so it just uses `channel` here